DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/3RicercaDeiProdotti.tex

```diff
@@ -53,7 +49,7 @@ \subsection{UC6 - Filtraggio prodotti della PLP} \label{UC6}
         \end{itemize}
         \item La pagina viene aggiornata con i prodotti che rispettano i filtri applicati
     \end{itemize}
-    \item \textbf{Scenario Alternativo:} (UC5.1) - Nel caso i filtri selezionati non diano un risultato verrà visualizzato un messaggio dove viene segnalato all'attore questo fatto.
+    \item \textbf{Estensioni:} (UC5.1) - Nel caso la combinazione di filtri impostati non dia risultati verrà visualizzato un messaggio dove viene segnalato all'attore questo fatto.
```

Comment:
L'estensione va messa in un itemize

Reply:
Inserire scenario alternativo nel quale va spiegato quando avviene l'errore (guardare come è stato fatto nella gestione delle categorie del venditore)

Reply:
Gli scenari alternativi io e Cristiano li abbiamo tolti perchè nell’esempio del prof sulle sue slide c’era solo l’estensione con la riga di spiegazione che non mettevamo in scenario alternativo sempre dentro all’estensione